.github/workflows/deploy-prod.yaml

```diff
@@ -297,6 +297,22 @@ jobs:
         upload_url: ${{ steps.create_release.outputs.upload_url }}
         asset_path: sbom/assets/*
 
+  set-current-version:
```

Comment:
should we make testgrid-run depend on this?

Reply:
done